rotkehlchen/db/history_events.py

```diff
@@ -31,9 +32,25 @@
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
-HISTORY_INSERT = """INSERT OR IGNORE INTO history_events(event_identifier, sequence_index,
-timestamp, location, location_label, asset, amount, usd_value, notes,
-type, subtype, counterparty, extra_data) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);"""
+HISTORY_BASE_ENTRY_INSERT = """INSERT OR IGNORE INTO history_events(event_identifier,
+ sequence_index, timestamp, location, location_label, asset, amount, usd_value, notes,
+ type, subtype) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);"""
+
+HISTORY_BASE_ENTRY_SELECT = 'SELECT * FROM history_events '
+
+HISTORY_BASE_ENTRY_SELECT_COUNT = 'SELECT COUNT(*) FROM history_events '
+
+EVM_EVENT_INSERT = """INSERT OR IGNORE INTO evm_events_info(identifier, protocol, product,
+address, extra_data) VALUES (?, ?, ?, ?, ?);"""
+
+EVM_EVENT_SELECT = """SELECT A.identifier, A.event_identifier, A.sequence_index, A.timestamp,
+A.location, A.location_label, A.asset, A.amount, A.usd_value, A.notes, A.type, A.subtype,
+B.protocol, B.product, B.address, B.extra_data
+ FROM history_events as A INNER JOIN evm_events_info as B ON A.identifier=B.identifier """
```

Comment:
What do you think? Lefteris had concerns about performance but based on the swap events that was a complex query that we were using before. Here we are just doing a join. Do you think it will improve the work on filters?

Reply:
A view is a helper sql script in the db, right?

If yes, then hmm... Does this `EVM_EVENT_SELECT` constant and others look badly-readable to you?

Reply:
Yes. I propose this thinking in the filters and how you would need to use them later, to avoid repeating all the time the join as this operation is not consuming and can go in a view. If you think is not such a big deal we can just go with joining everywhere